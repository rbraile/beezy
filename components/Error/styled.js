import styled from 'styled-components'

export const Message = styled.p`
  color: red;
  margin: 20px 0;
`
