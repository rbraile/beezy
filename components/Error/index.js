import React from 'react'
import { Message } from './styled'

const Error = ({ message }) => {
  return (
    <div>
      <Message>{message}</Message>
    </div>
  )
}

export default Error
