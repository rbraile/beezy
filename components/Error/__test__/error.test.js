import React from 'react'
import { mount, shallow } from 'enzyme'

import Error from '../index'

describe('TEST = Error', () => {
  test('TEST = Error = SNAPSHOT', () => {
    const wrapper = mount(<Error />)
    expect(wrapper).toMatchSnapshot()
  })

  test('TEST = Error = SNAPSHOT', () => {
    const wrapper = mount(<Error message='error 404' />)

    expect(
      wrapper
        .find('styled__Message')
        .find('p')
        .text()
    ).toBe('error 404')
  })
})

// styled__Message
