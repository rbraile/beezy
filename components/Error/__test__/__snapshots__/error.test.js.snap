// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TEST = Error TEST = Error = SNAPSHOT 1`] = `
<Error>
  <div>
    <styled__Message>
      <p
        className="styled__Message-sc-16tbsrm-0 laOTbw"
      />
    </styled__Message>
  </div>
</Error>
`;
